SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1201,6 +1212,16 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 		TimerManager.Tick(DeltaTime);
 	}
 
+	if (GetDefault<USpatialGDKSettings>()->bBatchSpatialPositionUpdates)
+	{
+		if ((Time - TimeWhenPositionLastUpdated) >= (1.0f / GetDefault<USpatialGDKSettings>()->PositionUpdateFrequency))
+		{
+			TimeWhenPositionLastUpdated = Time;
+
+			Sender->ProcessPositionUpdates();
```

Comment:
Do we want to batch all position updates together? Spiky traffic tends to have it's own detrimental affect (although I don't have a good intuition for what magnitude).

Reply:
That said: if we've measured that this is an improvement for the entity numbers we care about then great and we can make this more complicated later if those entity numbers increase.

Reply:
Yes, I think we deal with that potential optimisation separately. You can turn off the batching if it's problematic.